app/models/presentation.rb

```diff
@@ -26,6 +26,7 @@ def person_or_presenter
   end
 
   def associated_event_has_available_time
+    return unless duration
```

Comment:
Two things.

1.) We could avoid the `unless` keyword by going `return if duration.nil?` which is a bit more readable

2.) Duration should probably be a required field in the schema. I imagine if someone is creating a presentation we should be ensuring that it has a duration. That is unrelated to this PR and would be feature creep so this is fine but I'll talk it over with @brendandeere and maybe make a resulting issue.